utils/motorcontroller/MotorConfig.java

```diff
@@ -36,4 +36,10 @@ public enum Motor_Type {
             put(Motor_Type.CIM, MotorType.kBrushed);
         }
     };
+    // use: position (all we need is 2), velocity (all we need is 1), uncontrolled (turn it all down)
```

Comment:
I assume "2" and "1" here refer to the CAN frames.

A nit: that's specific to the SparkMaxes whereas Motor_Use isn't. So you could maybe generalize the comment to indicate that all we need is the position-related frames rather than saying "2", etc.

And more broadly: writing something like "all we need is 2" is the kind of comment that makes sense when you write it but likely won't make sense to anyone else or probably even to you if you look at this 6 months from now and haven't been thinking about CAN frames.